spec/src/main/asciidoc/appendix.adoc

```diff
@@ -22,12 +22,32 @@
 
 === Alternatives considered
 
-Jolokia JMX-HTTP bridge. Using this for application specific metrics would require those metrics
-to be exposed to JMX first, which many users are not familiar with.
+We addressed some significant questions while creating MicroProfile Metrics v5.0.
+
+*API or no API*
+
+In light of the increasing prevalence of developer use of APIs from Micrometer and OpenTelemetry, we considered whether we should continue to have a distinct API for MicroProfile Metrics. We decided to continue our path of providing an API for the following reasons:
+
+1. provides an easy-to-use metrics API for application developers
+2. provides continuity for the existing MicroProfile Metrics user community
+3. provides a MicroProfile-style API (for example, CDI-based annotations), and configurability (MicroProfile Config), for ease of adoption by MicroProfile users
+4. ensures compatibility across APIs within the same MicroProfile release
+
+We also considered feedback from an informal poll asking if users would use a MicroProfile Metrics API, given the other options.
```

Comment:
Maybe allude to the outcome of the poll, maybe something along the lines of:

We also considered feedback from an informal poll in which a majority of respondents said they would use a MicroProfile Metrics API, given the other options.